app/models/wizard/steps/import_date.rb

```diff
@@ -1,38 +1,68 @@
 module Wizard
   module Steps
     class ImportDate < Base
+      include ActiveRecord::AttributeAssignment
+
+      attribute :import_date, :date
+
       attribute 'import_date(3i)', :string
       attribute 'import_date(2i)', :string
       attribute 'import_date(1i)', :string
 
       validate :import_date_in_future
 
-      def import_date
-        session[:import_date]
+      STEP_ID = 1
```

Comment:
Do we want this to be a string?

Reply:
I don't think it'll get serialised as an integer

Reply:
So we'll be mixing types.

Reply:
Uhm sure yeah.